features/admin.agents.v1/pages/agents.tsx

```diff
@@ -0,0 +1,199 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AdvancedSearchWithBasicFilters } from "@wso2is/admin.core.v1/components/advanced-search-with-basic-filters";
+import { AppConstants } from "@wso2is/admin.core.v1/constants/app-constants";
+import { history } from "@wso2is/admin.core.v1/helpers/history";
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { ListLayout, PageLayout, PrimaryButton } from "@wso2is/react-components";
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { Icon } from "semantic-ui-react";
+import AgentList from "../components/agent-list";
+import AddAgentWizard from "../components/wizards/add-agent-wizard";
+import { useGetAgents } from "../hooks/use-get-agents";
+
+interface AgentPageProps extends IdentifiableComponentInterface {
```

Comment:
```suggestion
type AgentPageProps = IdentifiableComponentInterface
```

Fix in a follow-up PR

Reply:
Fixed with [45002d3](https://github.com/wso2/identity-apps/pull/8077/commits/45002d3cf827ceaebf840a4620fd85532f1b99e8)